tests/Microsoft.VisualStudio.ProjectSystem.Managed.UnitTests/ProjectSystem/Configuration/ProjectConfigurationDimensionProviderTestBase.cs

```diff
@@ -569,9 +569,9 @@ private static void AssertDefaultOrEmpty(BaseProjectConfigurationDimensionProvid
         {
             if (provider.DimensionDefaultValue is not null)
             {
-                Assert.Single(result);
-                Assert.Equal(provider.DimensionName, result.First().Key);
-                Assert.Equal(provider.DimensionDefaultValue, result.First().Value);
+                (string key, string value) = Assert.Single(result);
```

Comment:
Do we use var in this case or no?

Reply:
`var` is a forbidden keyword in these parts. 🤠😂

Reply:
In test code too? I noticed we don't use in product code. Also method above uses var.

Reply:
We only use var when the type is evident.

Our.editorconfig file is set up to highlight how we use var. You'll see hints in the IDE.